dio/lib/src/multipart_file.dart

```diff
@@ -44,9 +50,13 @@ class MultipartFile {
     this.filename,
     MediaType? contentType,
     Map<String, List<String>>? headers,
+    String? filePath,
+    List<int>? value,
```

Comment:
These fields seem unnecessary if the `restoreMultipartFile` is the method of `MultipartFile`, they might bring extra difficulties for users to understand why the class has such fields since we already have related. In order to avoid outside reference, you can use the constructor in `restoreMultipartFile` rather than `multipartFileFromPathSync`.

Reply:
Amazing! I just made this change, WDYT?